### Compute the backend chess engine's move action server.

# Goal
---
# Result
ChessMove move    # engine's fully qualified move
---
# Feedback
uint16  thinking  # engine progress "thinking" in ticks
